code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -1159,6 +1159,26 @@
 	construction_time = 12 SECONDS
 	category = list("Cyborg Upgrade Modules")
 
+/datum/design/borg_upgrade_storage_increaser
+	name = "Engineer Cyborg Upgrade (Storage Increaser)"
```

Comment:
```suggestion
	name = "Cyborg Upgrade (Storage upgrade)"
```
as stated on the item, its for all, not just medical. or apparently engineering as stated here..